docker-compose.yml

```diff
@@ -113,10 +113,10 @@ services:
       dockerfile: Dockerfile
       args:
         NGINX_CONFIG: "nginx-dev.conf"
-        REACT_APP_LOCAL_BACKEND: "True"
-        REACT_APP_API_URL: "http://localhost:8000"
-        REACT_APP_CHARTIS_URL: "http://localhost:7070"
-        REACT_APP_EDITOAST_URL: "http://localhost:8090"
+        REACT_APP_LOCAL_BACKEND: "${REACT_APP_LOCAL_BACKEND:-True}"
+        REACT_APP_API_URL: "${REACT_APP_API_URL:-http://localhost:8000}"
+        REACT_APP_CHARTIS_URL: "${REACT_APP_CHARTIS_URL:-http://localhost:7070}"
+        REACT_APP_EDITOAST_URL: "${REACT_APP_EDITOAST_URL:-http://localhost:8090}"
```

Comment:
Few things here:
- These variable must be moved to environment
- It's not really useful to get the use var env if it exists
- We can rename them (removing the prefix `REACT_APP_`)